shift/src/main/java/org/coursera/android/shift/StringListSelector.java

```diff
@@ -0,0 +1,55 @@
+/*
+ Copyright 2015 Coursera Inc.
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+     http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+limitations under the License.
+ */
+
+package org.coursera.android.shift;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+class StringListSelector {
```

Comment:
At first I tried having extending a generic ListSelector class but discovered that SnappyDB's kryo serialization can not handle the generic type and although it can insert it into the database, it can not properly retrieve it. To make other types of ListSelectors (int, float, etc.) we will have to keep this format.